### DISCUSSION TITLE
Login input still pops when already add ssh

### DISCUSSION THREAD

**Discussion by ghost** [extrawurst/gitui#1518] [Q&A] [answered]
Hello,

I have added GitHub SSH key in my local machine (macOS)

```
ssh -T git@github.com
Hi lodisy! You've successfully authenticated, but GitHub does not provide shell access.
```
and also run `ssh-add`, with `Identity added` before running GitUI

when attempt to push via GitUI, the login input still pops, is this normal?

I am using fish shell. I am using the same GitUI config file from Linux.

**Comment by extrawurst** [comment, marked as answer]
How is the repo url defined in the config?

**Reply by ghost** [reply]
Right, I forget to change that in the git config. Thanks!
